packages/fela-dom/src/dom/connection/createNode.js

```diff
@@ -5,6 +5,7 @@ import type { NodeAttributes } from '../../../../../flowtypes/DOMNode'
 
 export default function createNode(
   nodes: Object,
+  id: string,
```

Comment:
Same ordering thing here. Would like to keep it consistent and only add params at the end.

Reply:
Updated 👍